Library/Formula/id3lib.rb

```diff
@@ -52,6 +53,12 @@ class Id3lib < Formula
     sha256 "a7881dc25665f284798934ba19092d1eb45ca515a34e5c473accd144aa1a215a"
   end
 
+  # fixes Unicode display problem in easytag: see Homebrew/homebrew-x11#123
```

Comment:
it's not so much a display problem as it is an encoding issue: the tags get filled with erroneous sequences and opening the mp3s in an unrelated player such as VLC then displays mojibake'd info.